examples/aws-ebs-csi-driver-self-managed/helm_values/aws-ebs-csi-driver.yaml

```diff
@@ -0,0 +1,226 @@
+# Default values for aws-ebs-csi-driver.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
```

Comment:
we don't need to copy over the entire values file - we only want to show the minimal config required to get the driver up and running (and if that doesn't require additional/custom values then we can remove this file completely)